Examples/Python/src/GeometryBuildingGen1.cpp

```diff
@@ -0,0 +1,155 @@
+// This file is part of the Acts project.
+//
+// Copyright (C) 2021 CERN for the benefit of the Acts project
+//
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+#include "Acts/Geometry/CylinderVolumeBounds.hpp"
+#include "Acts/Geometry/CylinderVolumeHelper.hpp"
+#include "Acts/Geometry/Layer.hpp"
+#include "Acts/Geometry/LayerArrayCreator.hpp"
+#include "Acts/Geometry/LayerCreator.hpp"
+#include "Acts/Geometry/SurfaceArrayCreator.hpp"
+#include "Acts/Geometry/TrackingVolume.hpp"
+#include "Acts/Geometry/TrackingVolumeArrayCreator.hpp"
+#include "Acts/Geometry/VolumeBounds.hpp"
+#include "Acts/Plugins/Python/Utilities.hpp"
+
+#include <array>
+#include <memory>
+#include <unordered_map>
+#include <vector>
+
+#include <pybind11/pybind11.h>
+#include <pybind11/stl.h>
+
+namespace py = pybind11;
+using namespace pybind11::literals;
+
+namespace Acts::Python {
+void addGeometryBuildingGen1(Context &ctx) {
+  auto m = ctx.get("main");
+
+  using SurfacePtrVector = std::vector<std::shared_ptr<const Surface>>;
+
+  py::class_<Acts::Layer, std::shared_ptr<Acts::Layer>>(m, "Layer");
+
+  {
+    auto creator =
+        py::class_<Acts::LayerCreator>(m, "LayerCreator")
+            .def(py::init<Acts::LayerCreator::Config>())
+            .def("cylinderLayer",
```

Comment:
Can this be bound directly without the lambda?

Reply:
If I rember correctly I did that because full binding would require the binding of the types of some default arguments, that are not necessary currently...